packages/flutter/lib/src/gestures/monodrag.dart

```diff
@@ -48,7 +59,14 @@ typedef GestureDragCancelCallback = void Function();
 ///  * [PanGestureRecognizer], for drags that are not locked to a single axis.
 abstract class DragGestureRecognizer extends OneSequenceGestureRecognizer {
   /// Initialize the object.
-  DragGestureRecognizer({ Object debugOwner }) : super(debugOwner: debugOwner);
+  DragGestureRecognizer({
+    @required this.dragStartBehavior,
+    Object debugOwner,
+  })  : assert(dragStartBehavior != null),
+        super(debugOwner: debugOwner);
+
+  /// Configure the behavior of offsets sent to [onStart]
```

Comment:
nit: this is really trivial but please end sentences in documentation with a period.